monitor/alerts.js

```diff
@@ -13,10 +15,17 @@ const foreignProvider = new Web3.providers.HttpProvider(FOREIGN_RPC_URL)
 const web3Foreign = new Web3(foreignProvider)
 
 async function main() {
-  const { foreignDeposits, homeDeposits, homeWithdrawals, foreignWithdrawals } = await eventsInfo()
-
-  const xSignatures = foreignDeposits.filter(findDifferences(homeDeposits))
-  const xAffirmations = homeWithdrawals.filter(findDifferences(foreignWithdrawals))
+  const { foreignDeposits, homeDeposits, homeWithdrawals, foreignWithdrawals, bridgeMode } = await eventsInfo()
+
+  let xSignatures
+  let xAffirmations
+  if (bridgeMode === BRIDGE_MODES.ARBITRARY_MESSAGE) {
+    xSignatures = foreignDeposits.filter(processedMsgNotDelivered(homeDeposits))
```

Comment:
please consider either to change names `foreignDeposits`, `homeDeposits`, `homeWithdrawals` and `foreignWithdrawals` to more common (e.g. `foreignToHomeRequests`, `homeToForeignRequests`, `foreignToHomeConfirmations` and `homeToForeignConfirmations`) under this PR or create another issue to address it later.

Reply:
Thanks for suggestion, fixed https://github.com/poanetwork/tokenbridge/pull/199/commits/4370fd25b4f61c3920f5a4ba63d42afc3045d24e